Bootstrapping simplified along with some minor changes (#3)
sendinblue/raml-mock-server: README.md

# RAML Mock Server 

> ##### Brief introduction   

Reviewer: I would remove the heading since the next line is the introductory line.

Author: Done

> This application allows you to run a REST API mock sever which uses [RAML](https://raml.org/). 

Reviewer: ```suggestion
This application allows to run a REST API mock sever which uses the [RAML](https://raml.org/) format for endpoint definition. 
```

Author: Done

> # Demo

Reviewer: ```suggestion
## Demo
```

Author: Done

> The only file you need to run your API demo is **raml/api.raml**, which is used by the mock server to create endpoints.

Reviewer: ```suggestion
To run the demo the API definition file `raml/api.raml` is required which creates a mock server with several endpoints.
```

Author: Done, only changed `several endpoints` with `an endpoint` as we have single endpoint in `api.raml`

> api.raml contains one basic example of **GET** request and to explore more please check the RAML 
> [developers page](https://raml.org/developer).

Reviewer: ```suggestion
The file `api.raml` contains one basic example of `GET` request. Check the RAML [developer's page](https://raml.org/developer) to explore more.
```

Author: Done

> #### Execute below command to start the demo:

Reviewer: Looks like the level of the heading is too deep.

Author: Done, using bullet points

    $ docker-compose up -d
       
Go to [http://localhost:8000/api/](http://localhost:8000/api/) and 

> that's all your mock server is up and running. 

Reviewer: ```suggestion
Open [http://localhost:8000/api/] to ensure your mock server is up and running. 
```

Author: Done

> **ps:** Make sure to change the port in docker-compose.mock-server.yml, if it's already occupied.

Reviewer: I would not put a PS in the middle of the text.
```suggestion
Ensure coosing a different port in docker-compose.mock-server.yml in the case `8000` is already occupied.
```

Author: Done, changed `coosing` to `choosing` :P

> If you make any changes in api.raml, you'll have to execute the command below for changes to make effect.

Reviewer: ```suggestion
Ensure restarting the mock server on any change in `api.raml`. The following command can be used for that:
```

Author: Done

Author: Done

    $ docker-compose restart 

> # Integration

Reviewer: There should be only one Level 1 heading in the entire readme.
```suggestion
## Integration
```

Author: Done

> * Ensure `api.raml` file exists in your project (e.g. `raml/api.raml`).

Reviewer: ```suggestion
- Ensure the endpoint definition file and exists in the project and is located in a dedicated (mountable) folder, e.g. `raml/api.raml`.
```

Author: Done

> * Create `docker-compose.mock-server.yml` into root folder of your project with below code:

Reviewer: ```suggestion
- Create `docker-compose.mock-server.yml` in the root folder of the project. The file should contain the following definition. The mounted `raml` folder must be adjusted if needed.
```

Author: Done

```yaml
version: '3.4'
services:
  raml-mock-server:
    image: raml-mock-server
    ports:
    - "8000:80"
    volumes:
    - ./raml:/raml
```

> * To start/restart the mock server for your project, please execute:  

Reviewer: ```suggestion
* Start the mock server for your project by executing the following command:  
```

Author: Done

> ```-
> $ dcker-compose -f docker-compose-mock-server.yml up -d
> 
> $ docker-compose -f docker-compose-mock-server.yml restart
> ```

Reviewer: ```suggestion
```
$ docker-compose -f docker-compose-mock-server.yml up -d
```
- Restart the mock server on any change in `api.raml` file:
```
$ docker-compose -f docker-compose-mock-server.yml restart
```
```

Author: Done
